tests/integration/components/ember-notify/message-test.js

```diff
@@ -0,0 +1,40 @@
+/* jshint expr:true */
+import Ember from 'ember';
+import {
+  describeComponent,
+  it
+} from 'ember-mocha';
+import Notify from 'ember-notify';
+import hbs from 'htmlbars-inline-precompile';
+
+describeComponent(
+  'ember-notify/message',
+  'EmberNotifyMessageComponent | Integration',
+  {
+    integration: true
+  },
+  function() {
+    before(() => Notify.testing = true);
+    after(() => Notify.testing = false);
+
+    it('renders block version', function() {
+      const dummyMessage = Ember.Object.create({
+        text: 'Dummy text',
+        visible: true
+      });
+      this.set('message', dummyMessage);
+
+      // Template block usage:
+      this.render(hbs`
+        {{#ember-notify/message message=message as |message close|}}
+          <a {{action close}} class='close'>CLOSE</a>
+          <span class='message'>{{message.text}}</span>
+        {{/ember-notify/message}}
+      `);
+
+      expect(this.$().find('.message').text()).to.equal(dummyMessage.get('text'));
+      this.$().find('.close').click();
+      expect(dummyMessage.get('visible')).to.be.false;
```

Comment:
I'd like to see assertions here that confirm that the supplied block have indeed been used.